### ISSUE TITLE
The Symmetry of loss function

### ISSUE THREAD

**Issue by senLeee** [princeton-nlp/SimCSE#45]
Given a sentence xi, after dropout twice, we can get its representations of hi^z1, hi^z2.
In Equation(4) (Section 3 Unsupervised SimCSE), it seems you only use contrastive_loss(hi^z1, hi^z2) as the training objective, have you tried using 1/2*contrastive_loss(hi^z1, hi^z2) + 1/2*contrastive_loss(hi^z2, hi^z1) (i.e., the loss in paper ''A Simple Framework for Contrastive Learning of Visual Representations'')?

**Comment by gaotianyu1350** [comment]
Hi,

Good point! We have indeed tried this out and we found that it did not make a difference with regard to the performance.
